src/standard/utils.html

```diff
@@ -306,6 +306,18 @@
         }
       }
       return elt;
+    },
+
+    /**
+     * Convenience method for checking whether an element is in this element's
+     * shadow DOM tree;
+     *
+     * @method isShadowDomChild
+     * @param {HTMLElement=} node The element to be checked.
+     * @return {Boolean} true if node is in this element's shadow DOM tree.
+     */
+    isShadowDomChild: function(node) {
+      return (Polymer.dom(this).getOwnerRoot() !== Polymer.dom(node).getOwnerRoot());
```

Comment:
@samccone Don't need to guard: Polymer.dom(null) is well defined and returns `document` :)